/** (C) Copyright 2013, Applied Physical Sciences Corp., A General Dynamics Company
 **
 ** Gravity is free software; you can redistribute it and/or modify
 ** it under the terms of the GNU Lesser General Public License as published by
 ** the Free Software Foundation; either version 3 of the License, or
 ** (at your option) any later version.
 **
 ** This program is distributed in the hope that it will be useful,
 ** but WITHOUT ANY WARRANTY; without even the implied warranty of
 ** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ** GNU Lesser General Public License for more details.
 **
 ** You should have received a copy of the GNU Lesser General Public
 ** License along with this program;
 ** If not, see <http://www.gnu.org/licenses/>.
 **
 */

syntax = "proto2";
option optimize_for = SPEED;
option java_outer_classname = "ComponentDataLookupResponseContainer";
option java_package = "com.aphysci.gravity.protobuf";

package gravity;
message GravitySpdLogConfigPB
{
	optional string component_id = 1; 
	enum LoggerLevel
	{
		trace = 0;
		debug = 1;
		info = 2;
		warn = 3;
		error = 4;
		critical = 5;
		off = 6;
	} 			       
	optional LoggerLevel logger_level  = 2;
	enum LoggerType
	{
		GravityConsoleLogger = 0;
		GravityFileLogger = 1;
		ApplicationConsoleLogger = 2;
		ApplicationFileLogger = 3;
		ApplicationNetworkLogger = 4;

	}
	optional LoggerType logger_id = 3;
}